v2/futures/order_service.go

```diff
@@ -1086,3 +1086,121 @@ func (s *CreateBatchOrdersService) Do(ctx context.Context, opts ...RequestOption
 
 	return batchCreateOrdersResponse, nil
 }
+
+// ModifyBatchOrdersService handles batch modification of orders
+type ModifyBatchOrdersService struct {
+	c      *Client
+	orders []*ModifyOrderService
```

Comment:
```suggestion
	orders []*ModifyOrder
```
Please create a new type named `ModifyOrder` instead of `ModifyOrderService`, since the `xxxService` naming convention is confusing, and it's better not to rely on another Service